sdk/tables/azure-data-tables/tests/test_table_async.py

```diff
@@ -282,10 +278,10 @@ async def test_delete_table_with_existing_table(self, resource_group, location,
 
         # Assert
         self.assertIsNone(deleted)
-        # existing = list(ts.query_tables("TableName eq '{}'".format(table.table_name)))
+        # TODO: the AsyncPageIterator is not an iterable and this check fails
```

Comment:
It is an iterator, but it must be iterated asynchronously - so using it with `list()` will not work.